src/Entity/SettingInterface.php

```diff
@@ -150,4 +161,15 @@ public function isProject(): bool;
      * @return bool
      */
     public function isShared(): bool;
+
+    /**
+     * Specification:
+     * - Returns `true` if it just created.
+     * - Returns `false` if it already has existed.
+     *
+     * @api
+     *
+     * @return bool
+     */
+    public function isFirstInitialization(): bool;
```

Comment:
is it an important method? I'm not sure that we need it here.

Reply:
Yes i need to identify the first setting initialization to ask the user's email. On the second run it should be skipped. Usually `id` is used for that but we don't expose it

Reply:
How you are going to replace this email? in this case it's not possible

Reply:
if you need it move to interface in core like `SprykerSdk\Sdk\Core\Domain\Entity\ContextInterface`